### PR TITLE
Changed gather references to aether

### PR DESCRIPTION
**Pull request by realdouno**
- Changed all gather references to aether (in files and folder structure)  - [https://jira.ehealthafrica.org/browse/AET-2](https://jira.ehealthafrica.org/browse/AET-2)
- ODK module submission now using the common module ([This ticket](https://jira.ehealthafrica.org/browse/AET-45) is no longer needed - can be closed)

### CODE REVIEW

**Comment by admbtlr** [comment]
This looks great 👍 
(although admittedly I haven't read through the whole PR...)

But let's not merge it into 23 until 23 itself is ready to merge into Develop

**Review by quesebifurcan — CHANGES_REQUESTED** [review]
Great work, I just tried a test merge of this branch into a local aether-23 and all tests pass.

However, I'm getting

```
Invalid Login

You tried to login without providing information on the service you want to login to.

This page should not be accessed directly.
```

when attempting to log in to the browsable API here: http://core.aether.local:8000/

**Pull request closed without merging**
